### DISCUSSION TITLE
OpenDTU-GrafanaStack

### DISCUSSION THREAD

**Discussion by Kraego** [tbnobody/OpenDTU#916] [Ideas]
Hello,
I've started creating a docker stack to run grafana monitoring (mqqt broker - telegraf - influxdb2 - grafana).
See: https://github.com/Kraego/OpenDTU-Grafana-Stack 

The final goal is startup the whole thing with just setting the correct values in .env and run `docker-compose up -d`.

Currently it is working with some manual steps. **Contributions welcome!** 

The guide how to setup the whole thing without docker can be found here: https://github.com/Kraego/OpenDTU-Grafana-Howto/blob/main/OpenDTUGrafana.md

**Comment by movcmpret** [comment]
Hi,

unfortunately I didn't see this discussion earlier. 😅 So I did another, similar setup. 

https://github.com/movcmpret/opendtu-visualisation-docker

Hope it helps

**Reply by kyannik** [reply]
Hey thanks - I got it working so far. Could you elaborate on the steps required to actual display the data within Grafana? I am struggling to make this template work: https://grafana.com/grafana/dashboards/18819-opendtu-flux/?cmdf=grafana+dashboard+opendtu

**Reply by movcmpret** [reply]
Hi @kyannik, 
thanks for your feedback. I added an example dashboard:

https://github.com/movcmpret/opendtu-visualisation-docker/blob/github-main/example_dashboard.json
